app/models/katello/content_view_version.rb

```diff
@@ -73,8 +73,8 @@ def self.with_library_repo(repo)
 
     def self.for_version(version)
       major, minor = version.to_s.split('.')
-      query = where(:major => major)
-      query.where(:minor => minor) if minor
```

Comment:
I think this is a good question. The only place `for_version` is used is in the api (and thus the cli). I think the two factors would be:

1. Do we have to maintain backwards compatibility? If so, @johnpmitsch's solution is good.
2. What would a user who filters by "1" expect? Just 1.0 or 1.0, 1.1, etc? I'd think the former. To me 1 == 1.0 == 1.0.0 ....